R/calcNitrogenBNF.R

```diff
@@ -11,24 +11,21 @@
 #' }
 #'
 calcNitrogenBNF <- function(cellular = FALSE) {
-
   land <- calcOutput("LanduseInitialisation", aggregate = FALSE, cellular = TRUE)
   bnfRate <- calcOutput("NitrogenFixationRateNatural", aggregate = FALSE)
   bnf <- land * bnfRate
   bnf[, , c("crop", "urban")] <- 0
 
   if (!cellular) {
-    mapping <- toolGetMapping(name = "CountryToCellMapping.rds", where = "mrcommons")
-    bnf <- groupAggregate(data = bnf, query = mapping, from = "celliso", to = "iso", dim = 1)
-    bnf  <- toolCountryFill(bnf, fill = 0)
-  } else {
-    bnf <- toolCell2isoCell(bnf)
+    mapping <- data.frame(celliso = getItems(bnf, 1, full = TRUE),
+                          iso = getItems(bnf, if (dimExists("iso", bnf)) "iso" else 1.1, full = TRUE))
+    bnf <- toolAggregate(bnf, rel = mapping, from = "celliso", to = "iso")
```

Comment:
Does this work? Shouldn't this be 67k cells and therefore no longer celliso but coords?